webpack/InsightsHostDetailsTab/NewHostDetailsTab.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { push } from 'connected-react-router';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import SearchBar from 'foremanReact/components/SearchBar';
+import { Grid, GridItem } from '@patternfly/react-core';
+import InsightsTable from '../InsightsCloudSync/Components/InsightsTable';
+import RemediationModal from '../InsightsCloudSync/Components/RemediationModal';
+import Pagination from '../InsightsCloudSync/Components/InsightsTable/Pagination';
+import { INSIGHTS_SEARCH_PROPS } from '../InsightsCloudSync/InsightsCloudSyncConstants';
+import { fetchInsights } from '../InsightsCloudSync/Components/InsightsTable/InsightsTableActions';
+
+const NewHostDetailsTab = ({ hostName }) => {
+  const dispatch = useDispatch();
+  const withHostnameQuery = q => `hostname = ${hostName} AND ${q}`;
```

Comment:
Can `${q}` be wrapped with `()` ? as some queries can be wrong I think. for example `q = total_risk = 0 or total_risk=  1`

Reply:
thanks, added!